pygotham/manage/events.py

```diff
@@ -44,6 +44,10 @@ def run(self):
             event = Event()
             form.populate_obj(event)
 
+            if event.active:
+                now = arrow.utcnow().to('America/New_York').naive
```

Comment:
In the future, we should hoist all instances of timezone strings into a setting to allow conferences elsewhere to use the project.